Update content-style-guide.md (#3390)
fishtown-analytics/docs.getdbt.com: contributing/content-style-guide.md

## Links

Links embedded in documentation are about trust. Users trust that we will lead them to sites or pages related to their reading content. In order to maintain that trust, it's important that links are transparent, up-to-date, and lead to legitimate resources.

### Internal links

> We require root-relative _file_ paths (include the file extension, such as `.md`) or root-relative _URL_ paths (don't include `.md`) without the domain, docs.getdbt.com. We prefer this over using document-relative paths (relating to your current document, like one directory above it `../LinkedDocument`) because root-relative paths will work properly during local development and testing, and moving a document won't break the links it contains.  

Author: ```suggestion
We require root-relative _file_ paths (include the file extension, such as `.md`) or root-relative _URL_ paths (don't include `.md`) without the domain, docs.getdbt.com. We prefer this over using document-relative paths (relating to your current document, like one directory above it `../LinkedDocument`) because root-relative paths will work properly during local development and testing, and moving a document won't break the links it contains.   Refer to the [Docusaurus docs](https://docusaurus.io/docs/markdown-features/links) for more information.
```

> We require root-relative _file_ paths (include the file extension, such as `.md`) or root-relative _URL_ paths (don't include `.md`) without the domain, docs.getdbt.com. We prefer this over using document-relative paths (relating to your current document, like one directory above it `../LinkedDocument`) because root-relative paths will work properly during local development and testing, and moving a document won't break the links it contains.  

Author: ```suggestion
All internal links should use relative and not absolute paths. We construct these paths in relation to the content root, which is `[https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs)`, and they don't include the domain, https://docs.getdbt.com.

We require  either _file_ paths relative to the content root (these include the file extension, such as `.md`) or _URL_ paths relative to the content root (these don't include `.md`). We avoid paths relative to the document (for example, one directory above a document `../LinkedDocument`) because they won't work during local development and testing, and moving a document won't break the links it contains.  
```

## Links

Links embedded in documentation are about trust. Users trust that we will lead them to sites or pages related to their reading content. In order to maintain that trust, it's important that links are transparent, up-to-date, and lead to legitimate resources.

### Internal links

> All internal links should use relative and not absolute paths. We construct these paths in relation to the content root, which is`[docs.getdbt.com repository/website/docs](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs)`, and they don't include the domain, https://docs.getdbt.com.

Author: ```suggestion
All internal links should use relative and not absolute paths. We construct these paths in relation to the content root, which is`[_docs.getdbt.com repository_/website/docs](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs)`, and they don't include the domain, https://docs.getdbt.com.
```

> All internal links should use relative and not absolute paths. We construct these paths in relation to the content root, which is`[_docs.getdbt.com repository_/website/docs](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs)`, and they don't include the domain, https://docs.getdbt.com.

Author: ```suggestion
All internal links should use relative and not absolute paths. We construct these paths in relation to the content root, which is`[_docs.getdbt.com repository_/website/docs](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs)`.
```

> All internal links should use relative and not absolute paths. We construct these paths in relation to the content root, which is `[https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs)`, and they don't include the domain, https://docs.getdbt.com.

Author: ```suggestion
All internal links should use relative and not absolute paths. We construct these paths in relation to the content root, which is`[docs.getdbt.com repository/website/docs](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs)`, and they don't include the domain, https://docs.getdbt.com.
```

> We require  either _file_ paths relative to the content root (these include the file extension, such as `.md`) or _URL_ paths relative to the content root (these don't include `.md`). We avoid paths relative to the document (for example, one directory above a document `../LinkedDocument`) because they won't work during local development and testing, and moving a document won't break the links it contains.  

Reviewer: ```suggestion
We require  either _file_ paths relative to the content root (these include the file extension, such as `.md`) or _URL_ paths relative to the content root (these don't include `.md`). We avoid paths relative to the document (for example, one directory above a document `../LinkedDocument`) because they won't work during local development and testing, and moving a document won't break the links it contains.  Using the full URL will open the page in a new browser tab, so we avoid this for docs.getdbt.com links. 
```

> Markdown links in Docusaurus open in the same window rather than creating a new browser tab, but you can use HTML to open a link in a new tab.

Author: ```suggestion
Markdown links in Docusaurus open in the same window rather than creating a new browser tab, but you can use HTML or full URLs to open a link in a new tab.
```
